src/views/homepage/components/featured-content-grid/certifications-featured-card.module.css

```diff
@@ -14,11 +14,41 @@
 	position: absolute;
 	right: 0;
 	top: -64px;
+	width: 100%;
 }
 
 .certificationsCardContentText {
 	display: flex;
 	flex-direction: column;
 	gap: 24px;
-	margin-bottom: 70px;
+}
+
+.title {
+	composes: hds-font-weight-bold from global;
+	font-family: var(--token-typography-display-400-font-family);
+	font-size: var(--token-typography-display-400-font-size);
+	line-height: var(--token-typography-display-400-line-height);
+	margin: 0;
+	padding: 0;
+
+	@media (--dev-dot-tablet-down) {
+		font-family: var(--token-typography-display-300-font-family);
+		font-size: var(--token-typography-display-300-font-size);
+		line-height: var(--token-typography-display-300-line-height);
+	}
```

Comment:
⛏️  I might be looking at the wrong designs, and also I feel like the `600px - 1000px` viewport range is just hard to design for sometimes, but I'm seeing a significant difference in font size so wanted to flag in case it's something we want to tweak:

![CleanShot 2023-04-20 at 10 42 49@2x](https://user-images.githubusercontent.com/4624598/233402747-f5f54340-2652-4295-8b8b-d4839ce70c9f.png)